tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py

```diff
@@ -48,14 +48,18 @@ def tearDown(self):
         """At the end of test execution, it checks for the errors"""
         self.assertTrue(self.test_passed, msg=self.failcomment)
 
-    def take_screenshot(self):
+    def take_screenshot(self, browser=False, browserInstance=None):
```

Comment:
Is the browser=False argument really necessary? It seems to me that you could also just check if there is a browserInstance object.